internal/cmd/diag.go

```diff
@@ -34,3 +35,22 @@ func ExecInUvm(ctx context.Context, vm *uvm.UtilityVM, req *shimdiag.ExecProcess
 	err = cmd.Run()
 	return cmd.ExitState.ExitCode(), err
 }
+
+func ExecInHost(ctx context.Context, req *shimdiag.ExecProcessRequest) (int, error) {
+	if len(req.Args) == 0 {
+		return 0, errors.New("missing command")
+	}
+	cmdArgsWithoutName := []string{""}
+	if len(req.Args) > 1 {
+		cmdArgsWithoutName = req.Args[1:]
+	}
+	cmd := exec.Command(req.Args[0], cmdArgsWithoutName...)
+	err := cmd.Run()
```

Comment:
Worth doing `cmd.Output()` instead and logging the output on the failure case also?

Reply:
Changed to CombinedOutput

Reply:
Even better, I forgot there's both of those